docs/GettingStarted.md

```diff
@@ -95,9 +95,9 @@ This will build the following binaries:
                 and the EbpfCore and NetEbpfExt drivers to be loaded on a remote system to test.
 * `socket_tests.exe`: Application for testing the eBPF extension that implements the BPF_CGROUP_SOCK_ADDR program type and related attach types.
 
-and a few binaries just used for demo'ing eBPF functionality, as in the demo walkthrough discussed below:
+and a few binaries just used for demo'ing eBPF functionality, as in the demo walk-through discussed below:
```

Comment:
Some dictionaries do have walkthrough as unhyphenated:
* https://www.urbandictionary.com/define.php?term=walkthrough
* https://dictionary.cambridge.org/dictionary/english/walkthrough

See https://grammarhow.com/walkthrough-walk-through-or-walk-through/ for discussion that discusses both and says it's a stylistic choice.  I'd prefer reverting.   Any spell-check utility ought to have the ability to add words to the dictionary and this one should be added.

Reply:
tutorial.md still has changes to change "walkthrough" to "walk-through"